firebase-installations/src/main/java/com/google/firebase/installations/remote/FirebaseInstallationServiceClient.java

```diff
@@ -158,15 +162,17 @@ public InstallationResponse createFirebaseInstallation(
         writeFIDCreateRequestBodyToOutputStream(httpURLConnection, fid, appId);
 
         int httpResponseCode = httpURLConnection.getResponseCode();
+        requestLimiter.setNextRequestTime(httpResponseCode);
 
         if (httpResponseCode == 200) {
```

Comment:
This obviously seemed to have worked in the past, but maybe it's better we check for a successful code here?
I'm not sure, but I think the FIS server responds with `201` in case of a successful registatration.